### PR TITLE
Update give-mailbox-permissions-to-another-user.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about Full access- It does not include Sendas or behalf of and only has access to Read and manage.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3729

File: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md [hunk 1]
```diff
@@ -117,6 +117,10 @@ There are a few different ways you can access a mailbox once you've been given a
 
 6. Select **Save**.
 
+
+> [!NOTE]
+> Read and Modify access is also called as Fullaccess permissions when granting access through Exchange admin center. Full access does not grant **Send as** or **Send on Behalf**  permission.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md]:
```suggestion
> Read and Manage permissions are called Full Access permission when granting access through the Exchange admin center. Full Access permission does not grant **Send as** or **Send on Behalf**  permissions.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Coy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**